packages/components/inputs/search-select-input/src/search-select-input.story.js

```diff
@@ -0,0 +1,146 @@
+    const iconLeft = icons[select('iconLeft', ['', ...iconNames])];
+    const noOptionsMessage = () =>
+      text('No options message', 'No exact matches found');
+    const loadingMessage = () =>
+      text('Loading options message', 'Loading extact matches');
+    const optionType = select(
+      'Dropdown option style',
+      Object.values(SELECT_DROPDOWN_OPTION_TYPES),
+      SELECT_DROPDOWN_OPTION_TYPES.SINGLE_LINED_OPTION
+    );
+    return (
+      <React.Fragment>
+        <Section>
+          <Value
+            key={`${isMulti}`}
+            defaultValue={isMulti ? [] : undefined}
+            render={(value, onChange) => (
+              <div>
+                <SearchSelectInput
+                  horizontalConstraint={select(
+                    'horizontalConstraint',
+                    ['s', 'm', 'l', 'xl', 'scale'],
+                    'scale'
+                  )}
+                  optionType={optionType}
+                  hasError={boolean('hasError', false)}
+                  hasWarning={boolean('hasWarning', false)}
+                  aria-label={text('aria-label', '')}
+                  aria-labelledby={text('aria-labelledby', '')}
+                  isAutofocussed={boolean('isAutofocussed', false)}
+                  backspaceRemovesValue={boolean('backspaceRemovesValue', true)}
+                  id={text('id', '')}
+                  containerId={text('containerId', '')}
+                  isClearable={boolean('isClearable', false)}
+                  isDisabled={boolean('isDisabled', false)}
+                  isReadOnly={boolean('isReadOnly', false)}
+                  isMulti={isMulti}
+                  noOptionsMessage={noOptionsMessage}
+                  loadingMessage={loadingMessage}
+                  maxMenuHeight={number('maxMenuHeight', 220)}
+                  name={text('name', 'form-field-name')}
+                  onBlur={action('onBlur')}
+                  onChange={(event, info) => {
+                    action('onChange')(event, info);
+                    onChange(event.target.value);
+                  }}
+                  onFocus={action('onFocus')}
+                  onInputChange={action('onInputChange')}
+                  placeholder={text('placeholder', 'Select..')}
+                  tabIndex={text('tabIndex', '0')}
+                  tabSelectsValue={boolean('tabSelectsValue', true)}
+                  value={value}
+                  // Async props
+                  loadOptions={loadOptions}
+                  cacheOptions={boolean('cacheOptions', false)}
+                  iconLeft={
+                    iconLeft ? React.createElement(iconLeft) : undefined
+                  }
+                />
+                <div>
+                  <p>
+                    The options fetching uses the data (given below) to perform
+                    an exact match. It is case sensitive and it perfrom search
+                    based on{' '}
```

Comment:
```suggestion
                    In this example, our `loadOptions` function uses the data (given below) to perform
                    an exact match. It is case sensitive and it performs a search
                    based on{' '}
```

Just to make it clearer that this is just for the Storybook example, and not already set internally in the component.